hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveSyncClient.java

```diff
@@ -376,7 +376,9 @@ public void updateLastCommitTimeSynced(String tableName) {
         SerDeInfo serdeInfo = sd.getSerdeInfo();
         serdeInfo.putToParameters(ConfigUtils.TABLE_SERDE_PATH, basePath);
         table.putToParameters(HOODIE_LAST_COMMIT_TIME_SYNC, lastCommitSynced.get());
-        table.putToParameters(HOODIE_LAST_COMMIT_COMPLETION_TIME_SYNC, lastCommitCompletionSynced.get());
+        if (lastCommitCompletionSynced.isPresent()) {
+          table.putToParameters(HOODIE_LAST_COMMIT_COMPLETION_TIME_SYNC, lastCommitCompletionSynced.get());
+        }
```

Comment:
Checking if it's present shouldn't hurt but not sure if it's necessary. Setting it to null seems to have the same behavior as not setting it at all?

Reply:
I'm afrait if not present,  lastCommitCompletionSynced.get() would raise a NPE